docs/CNVs/CNV-methods.tex

```diff
@@ -773,12 +773,31 @@ \subsubsection{The model}
 
 A natural choice for $\bar{\Sigma}_t$ will be made apparent soon. In the meantime, we notice that the $\vSigma_s$ always appears in addition to $\vPsi$ such that the specific choice of $\bar{\Sigma}_t$ may seem to be immaterial: if $\vPsi^*$ is the MLE estimate obtained using the choice $\bar{\vSigma}^*$, we may infer that $\vPsi^* + \bar{\vSigma}^* - \bar{\vSigma}'$ is the MLE estimate had we used $\bar{\vSigma}'$ instead. However, the constraint $\Psi_t > 0$ implies that $\Psi^*_t + \bar{\Sigma}_t > 0$, such that our choice of $\bar{\Sigma}_t$ effectively sets a lower bound on the unexplained variance.\\
 
+\subsubsection{Estimating the mean read depth $d_s$}\label{sec:read_depth_estimation}
+Let us define $\lambda_{st} \equiv \PP_{st} c_{st} e^{b_{st}}$ as the total multiplicative bias per sample per target. Assuming that $\lambda_{st}$ is sharply peaked around its mode, the MLE for $d_s$ can be easily found by approximating the Poisson distributions with Gaussian like before:
+\begin{align}
+P(\vn_s | d_s \boldsymbol{\lambda}_s) &= \prod_{t|M_{st} \neq 0} \mathrm{Poisson}(n_{st} | d_s \lambda_{st}) \simeq \prod_{t|M_{st} \neq 0} \norm(n_{st}|d_s \lambda_{st}, d_s \lambda_{st})\nonumber\\
+&\propto \prod_t \left[\frac{1}{\sqrt{d_s \lambda_{st}}}\exp\left(-\frac{(n_{st} - d_s \lambda_{st})^2}{2 d_s \lambda_{st}}\right)\right]^{M_{st}}.
+\end{align}
+Replacing $\lambda_{st}$ with its mode, the log likelihood of read depth is given as:
+\begin{equation}
+\log P(\vn_s | d_s) = -\frac{1}{2} \sum_t M_{st}\left[\log d_s + \frac{1}{d_s \lambda_{st}}\,(n_{st} - d_s\lambda_{st})^2\right] + \mathrm{const.}
+\end{equation}
+Maximizing yields a quadratic equation with one acceptable solution:
+\begin{equation}
+d_s = \frac{\sqrt{4\llangle \vn^2 \rrangle_s \llangle \boldsymbol{\lambda}^2 \rrangle_s + \llangle \boldsymbol{\lambda} \rrangle_s^2} - \llangle \boldsymbol{\lambda} \rrangle_s}{2 \llangle \boldsymbol{\lambda}^2 \rrangle_s},
```

Comment:
Maximizing Eq 61 wrt `d` I get something slightly different:
[sqrt( 1 + 4 <<lambda>><<n^2/lambda>>) - 1]/[2 <<lambda>>].  Your result follows if you multiply throughout by `\lambda` in an intermediate step, but I believe this can't be done since `\lambda` depends on `t`